exercicios/para-casa/vetSystem.js

```diff
@@ -0,0 +1,111 @@
+function HistoricoMedico() {
+  this.consultas = [];
+  this.vacinas = [];
+  this.procedimentos = [];
+}
+
+HistoricoMedico.prototype.adicionarConsulta = function adicionarConsulta(data) {
+  this.consultas.unshift(data);
+  console.log(`A consulta realizada em ${data} foi adicionada ao histórico médico.`);
+}
+
+HistoricoMedico.prototype.adicionarVacina = function adicionarVacina(vacina) {
+  this.vacinas.push(vacina);
+  console.log(`A vacina ${vacina} foi adicionada ao histórico médico.`);
+}
+
+HistoricoMedico.prototype.adicionarProcedimento = function adicionarProcedimento(tipo, motivo) {
+  this.procedimentos.push({tipo, motivo});
+  console.log(`O procedimento ${tipo} foi adicionado ao histórico médico pelo motivo: ${motivo}`);
+}
+
+function Gato(nome, idade, cor, castrado) {
+  this.nome = nome;
+  this.idade = idade;
+  this.cor = cor;
+  this.castrado = castrado;
+  this.historico = new HistoricoMedico();
+}
+
+Gato.prototype.miar = function miar() {
```

Comment:
No meu eu fiz a mesma condição para os dois, se for gato mia e else para cachorro.